tasks/README.md

```diff
@@ -84,7 +86,8 @@ The preferred style of the keys should be `snake_case`.
   "implementations" : [
     {"name": "foo.sh", "requirements": ["shell"]},
     {"name": "foo.ps1", "requirements": ["powershell"]}
-  ]
+  ],
+  "files": ["my_module/lib/puppet_x/helpful_code.rb", "my_util/file/util/"]
```

Comment:
We should use `files` instead of `file`, as `files` is the directory actually available as a mount point.